lib/modules/userHighlight.js

```diff
@@ -186,12 +186,12 @@ modules['userHighlight'] = {
 					hoverColor: this.options.firstCommentColorHover.value
 				}
 			};
-			if (this.options.highlightOP.value) {
-				this.highlight('submitter');
-			}
 			if (this.options.highlightFriend.value) {
```

Comment:
In the screenshot the styles are ordered top-to-bottom in order of highest priority to lowest, it's from DevTools which re-orders selectors for convenience. If you were to look at the raw CSS it should look like:

```
.friend {}
.submitter {}
.moderator {}
.admin {}.
```

The if statements should be in that order too. Sorry for the confusion.